src/nupic/engine/Network.cpp

```diff
@@ -980,6 +980,11 @@ void Network::resetProfiling()
     regions_.getByIndex(i).second->resetProfiling();
 }
 
+void Network::addCustomRegionPackage(const char * path)
```

Comment:
I think if you add static to a function in the .cpp file, it makes it a static free function, so we wouldn't be able to use it in other files.